src/Gui/Windows/Forms/DecompilerMenus.cs

```diff
@@ -0,0 +1,535 @@
+#region License
+/* 
```

Comment:
This file should not be under source control

Reply:
Exact. It's back ever and ever….. I've removed it.